docs/ref/django-admin.txt

```diff
@@ -418,6 +418,14 @@ models that may have been removed since the fixture was originally generated.
 The :djadminopt:`--app` option can be used to specify a single app to look
 for fixtures in rather than looking through all apps.
 
+The :djadminopt:`--exclude` option may be provided to prevent specific
+applications or models (specified as in the form of ``app_label.ModelName``)
+from being loaded.  You can also mix application names and model names.
```

Comment:
Docs should mention the "(use multiple --exclude to exclude multiple apps/models)" part.